### DISCUSSION TITLE
G2 on Lathe Mode Get err.33

### DISCUSSION THREAD

**Discussion by Reprosense** [grblHAL/STM32F4xx#153] [Q&A]
hi everyone , i just try to test G2 on my lathe and show an err.33 on my code , can i get any information what could make it happen , here is my gcode and my video

https://github.com/grblHAL/STM32F4xx/assets/135573322/d0d1b971-16fc-4cd1-b7c3-83ccdebb5345

[1. TUTUP PENTIL VESTIC V2.txt](https://github.com/grblHAL/STM32F4xx/files/13776786/1.TUTUP.PENTIL.VESTIC.V2.txt)

**Comment by terjeio** [comment]
You will find more information if you search for [grbl error 33](https://duckduckgo.com/?t=ffab&q=grbl+error+33&ia=web).
